src/bp/core/services/auth/auth-service.ts

```diff
@@ -41,7 +41,13 @@ export default class AuthService {
   }
 
   async findUserByEmail(email: string, selectFields?: Array<keyof AuthUser>): Promise<AuthUser | undefined> {
-    return this.findUser({ email }, selectFields)
+    const user = await this.findUser({ email }, selectFields)
+    if (user) {
+      const superAdmins = (await this.configProvider.getBotpressConfig()).superAdmins
+      user.isSuperAdmin = superAdmins.includes(email)
```

Comment:
I would have created a public `assertSuperAdmin(email)` and call that when the user tries to access a super admin section but I'm fine with it either way 👍

Reply:
well ... check the previous file .. `needSuperAdminAccess` isn't it what it does ?